### ISSUE TITLE
not all Utilities::MPI::sum() are shown in doxygen

### ISSUE THREAD

**Issue by davydden** [dealii/dealii#4415]
just noticed that `Utilities::MPI::sum (const SymmetricTensor<rank,dim,Number> &local,...)` is not shown in the generated doxygen https://www.dealii.org/developer/doxygen/deal.II/namespaceUtilities_1_1MPI.html . 

Don't see anything wrong with the header that would confuse doxygen...

**Comment by bangerth** [comment]
The same is true for the function taking `Tensor<...>` as argument.

**Comment by bangerth** [comment]
But the function is listed here: https://www.dealii.org/developer/doxygen/deal.II/classSymmetricTensor.html#a6c9d8cfe285d568676bb6e308146c147
This is because of the `@relates` statement in the documentation:
```
    /**
     * Perform an MPI sum of the entries of a symmetric tensor.
     *
     * @relates SymmetricTensor
     */
    template <int rank, int dim, typename Number>
    SymmetricTensor<rank,dim,Number>
    sum (const SymmetricTensor<rank,dim,Number> &local,
         const MPI_Comm &mpi_communicator);
```
It's a good question whether the `@relates` statement is useful in this context. I wished doxygen would just show the function in both places...

**Comment by davydden** [comment]
> But the function is listed here:

but the `Utilities::MPI::` namespace is gone. So one would really need to guess where those functions are.

> It's a good question whether the @relates statement is useful in this context

i would remove those to keep the functions listed under `Utilities::MPI`. @dealii/dealii opinions?

**Comment by bangerth** [comment]
I agree.

**Comment by masterleinad** [comment]
Fixed by #5968.
